src/giskardpy/plugin_post_processing.py

```diff
@@ -0,0 +1,73 @@
+import giskardpy.identifier as identifier
+from giskardpy.plugin import GiskardBehavior
+from giskardpy.exceptions import InsolvableException
+from py_trees import Status
+import numpy as np
+from giskardpy.plugin_interrupts import detect_wiggling
+
+
+class PostProcessing(GiskardBehavior):
+
+    def __init__(self, name):
+        super(PostProcessing, self).__init__(name)
+        #GiskardBehavior.__init__(self, name)
+
+    def setup(self, timeout):
+        #return True
+        return super(PostProcessing, self).setup(timeout)
+
+    def initialise(self):
+        super(PostProcessing, self).initialise()
+
+    def update(self):
+        e = self.get_blackboard_exception()
+        if isinstance(e, InsolvableException):
+            return Status.FAILURE
+
+        if not self.check_reachability_xdot():
+            self.raise_to_blackboard(InsolvableException(u'not all soft constraints were satisfied'))
+            return Status.FAILURE
+
+        if self.final_wiggle_detection():
+            self.raise_to_blackboard(InsolvableException(u'endless wiggling detected'))
+            return Status.FAILURE
+
+
+        return Status.SUCCESS
+
+
+    def check_reachability_xdot(self):
+        num_joint_constraints = len(self.get_robot().controlled_joints)
+        H = self.get_god_map().safe_get_data(identifier.H)
+        b_mask = np.array(H.sum(axis=1) != 0)[num_joint_constraints:]
```

Comment:
This mask stuff is the same I use in the other `qp_problem_builder` right? Can you please turn this into some general functions in `utils.py`, that is used by both?